components/LiteMintNFT.vue

```diff
@@ -111,6 +122,28 @@ const formError = computed(() => {
 })
 
 const isFormValid = computed(() => !formError.value?.length)
+const bookName = computed(() => localISCNData.value?.contentMetadata?.name || '')
+const imagePreviewUrl = computed(() => {
+  const { ARWEAVE_ENDPOINT } = useRuntimeConfig().public
+  if (!formState.imageUrl) {
+    return null
+  }
+
+  const url = formState.imageUrl
+  if (url.startsWith('ar://')) {
+    return `${ARWEAVE_ENDPOINT}/${url.replace('ar://', '')}`
+  }
+
+  if (url.startsWith('ipfs://')) {
+    return `https://ipfs.io/ipfs/${url.replace('ipfs://', '')}`
+  }
```

Comment:
Seems more natural to use `url.replace('ar://', ${ARWEAVE_ENDPOINT}/)` and similarly `url.replace('ipfs://', 'https://ipfs.io/ipfs/')`